plugin/trino-geospatial/src/test/java/io/trino/plugin/geospatial/TestGeoSpatialQueries.java

```diff
@@ -47,6 +47,15 @@ protected QueryRunner createQueryRunner()
         }
     }
 
+    @Test
+    public void testDistinctGeometry()
+    {
+        assertThat(query("select distinct geom from (select st_geometryfromtext('POINT (-90 38.99)') as geom UNION all select st_geometryfromtext('POINT (-90 38.99)') as geom)"))
```

Comment:
Capitalize the SQL keywords. Also, use VALUES instead of UNION ALL between two queries to synthesize the base data, and format it as follows for readability:

```java
assertThat(query("""
                 SELECT DISTINCT ST_GeometryFromText(point)
                 FROM (VALUES 'POINT (-90 38.99)', 'POINT (-90 38.99)') t(point)
                 """))
        .result().matches(MaterializedResult.resultBuilder(getSession(), GEOMETRY)
                .row("POINT (-90 38.99)")
                .build());
```